components/omega/doc/userGuide/Logging.md

```diff
@@ -0,0 +1,79 @@
+(omega-user-logging)=
```

Comment:
As above, this needs to be added to `index.md`, probably at the end for now:
```
...
userGuide/QuickStart
userGuide/OmegaBuild
userGuide/DataTypes
userGuide/Logging
...
```

Reply:
Added logging in index.md. The fix will be on next commit.